src/site/content/en/blog/constructable-stylesheets/index.md

```diff
@@ -108,7 +109,7 @@ ignored with a warning as of Chrome 84.
 ## Using constructed stylesheets
 
 The second new feature introduced by Constructable StyleSheets is an
-[adoptedStyleSheets](https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets)
+[adoptedStyleSheets](https://drafts.csswg.org/cssom/#extensions-to-the-document-or-shadow-root-interface)
```

Comment:
https://www.w3.org/TR/cssom-1/#extensions-to-the-document-or-shadow-root-interface